_dashboards/visualize/area.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+title: Area chart
+parent: Visualize
+nav_order: 5
+---
+
+# Area chart
+
+Area charts can be created by searching OpenSearch, extracting data, and processing the results and can be added to a dashboard. An area chart is a line chart with the area between the line and the axis shaded with a color, and is a primary visualization type used to display time series data. You can create area charts in Dashboards using the Area visualization type or using the TSVB, Vega, or VisBuilder visualization tools. For this tutorial, you'll use the Area visualization type.
+
+# Create an area chart
+
+In this tutorial you'll create a simple area chart in the [OpenSearch Dashboards playground](https://playground.opensearch.org/app/home#/) using sample data.
+
+1. Select **Visualize** from the menu.
+2. Select **Create visualization**.
+3. Select **Area** and then select **opensearch_dashboards_sample_data_flights** for source.
+4. Select **+ Add** => **X-axis** in **Buckets**.
```

Comment:
Think you can also outline how to changethe y axis to represent other aggregations with a link to the [aggregations doc page](https://opensearch.org/docs/2.4/opensearch/aggregations/)